src/FSharp.Core/list.fsi

```diff
@@ -1411,6 +1584,9 @@ module List =
 
     /// <summary>Builds a new collection whose elements are the results of applying the given function
     /// to each of the elements of the collection. The integer index passed to the
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(n)
```

Comment:
Here the added complexity information should be included at the end of the summary and not in the middle of a sentence